src/main/java/org/opentripplanner/model/MultiModalStation.java

```diff
@@ -6,7 +6,12 @@
 import java.util.stream.Collectors;
 
 
-// TODO OTP2 MMSP - JavaDoc
+/**
+ * The next level grouping of stops above Station. Equivalent to NeTEx multimodal StopPlace. As
+ * a NeTEx StopPlace (Station) only supports a single transit mode, you are required to group
+ * several Stations together using a MultiModalStation in order to support several modes. This
+ * entity is not part of GTFS.
+ */
```

Comment:
The primary terms used should be the "GTFS" terminoloy, even for this class that is only relevant for Netex. Put Netex terms in `()`:
> As a NeTEx StopPlace (Station) only
 change to:
> As a Station (StopPlace, NeTEx) only

This is a minor thing, but we should try to be consequent.